appinfo/Migrations/Version20180123143343.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ *
+ * @author Semih Serhat Karakaya <karakayasemi@itu.edu.tr>
+ * @copyright Copyright (c) 2018, ownCloud GmbH
+ * @license GPL-2.0
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+namespace OCA\password_policy\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use OCP\Migration\ISchemaMigration;
+
+class Version20180123143343 implements ISchemaMigration {
+
+	/** @var  string */
+	private $prefix;
+
+	public function changeSchema(Schema $schema, array $options) {
+		$this->prefix = $options['tablePrefix'];
+		if (!$schema->hasTable("{$this->prefix}user_password_history")) {
+			$table = $schema->createTable("{$this->prefix}user_password_history");
```

Comment:
no auto increment id column - afaik this is necessary for the mapper to operate properly